great-idea/index.html

```diff
@@ -1,23 +1,78 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
   <title>Great Idea!</title>
-
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+        rel="stylesheet">
+  <link href="css/index.css"
+        rel="stylesheet">
 </head>
-
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
-  </body>
-</html>
\ No newline at end of file
+<body>
+  <div class="container">
+    <header>
+      <nav>
+        <a href="#">Services</a> <a href="#">Product</a> <a href="#">Vision</a> <a href="#">Features</a> <a href="#">About</a> <a href="#">Contact</a> <img alt="Great Idea! Company logo."
+             class="logo"
+             src="img/logo.png">
+      </nav>
```

Comment:
```suggestion
      <nav>
        <a href="./services.html">Services</a> 
        <a href="#">Product</a> 
        <a href="#">Vision</a> 
        <a href="#">Features</a> 
        <a href="#">About</a> 
        <a href="#">Contact</a> 
        <img alt="Great Idea! Company logo."
             class="logo"
             src="img/logo.png">
      </nav>
```

Reply:
Try lining up your navs like this so it's easier to read and work with. Also, be sure to include the filepath to your other pages in your <a> tags. Your services page should link back home as well.